starknet-e2e-test/ethereum_e2e.rs

```diff
@@ -0,0 +1,46 @@
+extern crate starknet_e2e_test;
+
+use std::time::Duration;
+
+use madara_runtime::opaque::Block;
+use madara_test_runner::node::MadaraTempDir;
+use madara_test_runner::{MadaraArgs, MadaraRunner, Settlement};
+use mc_settlement::ethereum::StarknetContractClient;
+use mc_settlement::{SettlementProvider, StarknetState};
+use rstest::rstest;
+use starknet_e2e_test::starknet_sovereign::StarknetSovereign;
+use tokio::time::sleep;
+
+#[rstest]
+#[tokio::test]
+async fn madara_with_sovereign_core_contract_on_ethereum() -> Result<(), anyhow::Error> {
```

Comment:
It tests that it connect correctly?  And shares data?
Maybe a name a lil more descriptive could help